src/Webfactory/Slimdump/SlimdumpCommand.php

```diff
@@ -29,6 +30,13 @@ protected function configure()
                 InputArgument::IS_ARRAY | InputArgument::REQUIRED,
                 'Configuration files (at least one).'
             )
+            ->addOption(
+                'buffer-size',
+                'b',
+                InputOption::VALUE_OPTIONAL,
+                'Maximum buffer for the database connected to',
```

Comment:
I'd rather rephrase this along the lines of "maximum length of a single SQL statement generated"...? Does that better describe the purpose?

Reply:
I will add to the documentation that this is optional.